types/agora-rtc-sdk/index.d.ts

```diff
@@ -1,314 +1,3329 @@
-// Type definitions for agora-rtc-sdk 2.3
+// Type definitions for agora-rtc-sdk 2.6
 // Project: https://github.com/AgoraIO/web-archive#readme
 // Definitions by: Menthays <https://github.com/menthays>
+//                 Derek Lu <https://github.com/scaret>
+//                 Wangjie <https://github.com/wangjie-agora>
+//                 Yu SiCheng <https://github.com/ysclyy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-/*~ If this module is a UMD module that exposes a global variable 'myLib' when
- *~ loaded outside a module loader environment, declare that global here.
- *~ Otherwise, delete this declaration.
+/**
+ * AgoraRTC is the entry point for all the methods that can be called in Agora Web SDK.
+ *
+ * You can use the AgoraRTC methods to create {@link Client} and {@link Stream} objects.
+ *
+ * Other methods of the AgoraRTC object check for system requirements and set up error logging.
  */
-export as namespace AgoraRTC;
+declare namespace AgoraRTC {
```

Comment:
A cursory glance at the first line of the main file of the package shows that the `agora-rtc-sdk` is a UMD module and can be loaded either via a global when run as a script on the web, or via `import` or `require`. This change makes it a global only and prevents it from being loaded as a module. It is possible this could break other callers.

Reply:
My fault. I will fix this. Thanks a lot.